src/dialog/overflow-menu/overflow-menu.directive.ts

```diff
@@ -51,6 +51,25 @@ export class OverflowMenuDirective extends DialogDirective {
 
 	onDialogInit() {
 		this.dialogConfig.content = this.ibmOverflowMenu;
+		this.dialogService.isClosed.subscribe(() => this.updateOpacity());
+	}
+
+	open() {
+		super.open();
+		this.updateOpacity();
+	}
+
+	toggle() {
+		super.toggle();
+		this.updateOpacity();
+	}
+
+	updateOpacity() {
+		if (this.elementRef.nativeElement.getAttribute("aria-expanded") === "true") {
+			this.elementRef.nativeElement.style.opacity = 1;
```

Comment:
Weighing in - Wondering if team wants CSS with attribute selector?

Reply:
Well this should be something that needs to be set in the `"bx--overflow-menu--open"` class, which is currently not doing anything

Reply:
So I'm not sure why this needs to be applied in the first place? Is the overflow menu getting `opacity: 0` in the table? if so why?

Reply:
Well the design is to have the menu item only appear when you hover on it and to only stay visible when the menu is open